lib/osf-components/addon/components/cedar-metadata-editor/template.hbs

```diff
@@ -1,13 +1,20 @@
-<div>
-    <cedar-embeddable-editor
-        config={{this.cedarConfig}}
-        templateObject={{@templateObject}}
-        instanceObject={{@instanceObject}}
-    >
-    </cedar-embeddable-editor>
-    <Button
-        {{on 'click' this.save}}
-    >
-        {{ t 'cedar.editor.save-button' }}
-    </Button>
+<div local-class='cedar-add-container'>
+    <div local-class='cedar-container'>
+        <cedar-embeddable-editor
+            local-class='cedar-editor'
+            config={{this.cedarConfig}}
+            templateObject={{@templateObject}}
+            instanceObject={{@instanceObject}}
+        />
+    </div>
+    <div local-class='action-container'>
+        <Button
+            data-test-save-button
+            aria-label= {{ t 'cedar.editor.save-button' }}
```

Comment:
Please remove this aria-label as the button already has visible text. The aria-label and the visible text can get mismatched as with the other ones, so best to avoid that possibility entirely